jerry-core/debugger/debugger.h

```diff
@@ -19,6 +19,16 @@
 #include "debugger-ws.h"
 #include "ecma-globals.h"
 
+#ifdef HAVE_TIME_H
+#define __USE_POSIX199309
+#include <time.h>
+#undef __USE_POSIX199309
+#elif defined (HAVE_UNISTD_H)
+#define __USE_MISC
+#include <unistd.h>
+#undef __USE_MISC
+#endif
```

Comment:
several comments:
- all these should go into the source file, not in this header
- the double-underscore defines are very suspicious. usually, you should not touch any of them. please, have a look at https://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html . most probably `_XOPEN_SOURCE` is the macro you are looking for. (also note that "You should define these macros by using ‘#define’ preprocessor directives at the top of your source code files. These directives must come before any #include of a system header file.") (and it does not neet to be undefined IMO)
- don't forget adding comments to preproc directives